src/MyFTP/MyFTPClient/Client.cs

```diff
@@ -0,0 +1,84 @@
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace MyFTPClient;
+
+public class Client
+{
+    private int port;
+    private IPAddress ip;
+    private CancellationToken token;
+
+    public Client(IPAddress ip, int port)
+    {
+        this.port = port;
+        this.ip = ip;
+        this.token = new CancellationToken();
+    }
+
+    /// <summary>
+    /// Listing of file by specified path
+    /// </summary>
+    /// <param name="path">Path on server to get listing on </param>
+    /// <returns>List of files and info about type</returns>
+    /// <exception cref="FileNotFoundException"> Specified path doesn't exist</exception>
+    public async Task<List<(string, bool)>> List(string path)
```

Comment:
```suggestion
    public async Task<List<(string Name, bool IsDir)>> List(string path)
```
Чтобы вызывающему было понятно, что за значения ему возвращают